systems/plants/RigidBodyManipulator.h

```diff
@@ -294,10 +294,21 @@ class DLLEXPORT_RBM RigidBodyManipulator
   Eigen::Matrix<typename Derived::Scalar, Derived::RowsAtCompileTime, Eigen::Dynamic> transformVelocityMappingToPositionDotMapping(
       const Eigen::MatrixBase<Derived>& mat, const std::vector<int>& joint_path);
 
-  template <typename Derived>
-  Eigen::Matrix<typename Derived::Scalar, Derived::RowsAtCompileTime, Eigen::Dynamic> compactToFull(
-      const Eigen::MatrixBase<Derived>& compact, const std::vector<int>& joint_path, bool in_terms_of_qdot);
-
+template <typename Derived>
+Eigen::Matrix<typename Derived::Scalar, Derived::RowsAtCompileTime, Eigen::Dynamic> compactToFull(
```

Comment:
i don't actually have any idea what this method does given the method name.  add comment?